packages/world/ts/config/v2/world.test.ts

```diff
@@ -194,7 +194,6 @@ describe("defineWorld", () => {
       systems: {},
       excludeSystems: [],
       deploy: {
-        customWorldContract: "(undefined)",
```

Comment:
this was in output but wasn't usable in input (was `never`) so I am cleaning it up here